next.config.mjs

```diff
@@ -7,9 +7,6 @@ import { withSentryConfig } from '@sentry/nextjs'
  **/
 const config = {
   swcMinify: true,
-  images: {
-    formats: ['image/avif', 'image/webp'],
-  },
```

Comment:
I figured out why images weren't loaded properly in browsers other than Chrome within Percy: because we were generating optimized images in AVIF format. I've disabled such optimization, so they are generated in Webp and it's working for all browsers in Percy.

I still need to figure out why Firefox doesn't load fonts properly, but that's a minor issue. We can go ahead now.

Reply:
> I still need to figure out why Firefox doesn't load fonts properly, but that's a minor issue. We can go ahead now.

I use Firefox on my computer and the homepage looks like the Percy snapshots for the Chrome, Safari and Edge browsers.

It looks to me that this is a problem with running Firefox within Percy somehow?

Reply:
Yes, it's a problem within Percy. We can go ahead and merge this branch tho, then we will fix Firefox screenshots when we figure out the issue with support's help.

Reply:
Sounds good to me!